themes/edsonarios/edsonarios.theme.sh

```diff
@@ -0,0 +1,19 @@
+#! bash oh-my-bash.module
+# Background: #333333
```

Comment:
Could you elaborate on this line? Maybe it means that this theme assumes the terminal background color to be `#333333`, but this is merely my guess. It is better to explicitly say what it actually means rather than expecting extra guess work from the readers.

Reply:
Done and comment on its use